packages/insomnia-inso/src/jest/setup.ts

```diff
@@ -1 +1 @@
-process.env.DEFAULT_APP_NAME = process.env.DEFAULT_APP_NAME || 'insomnia-app';
+process.env.DEFAULT_APP_NAME = process.env.DEFAULT_APP_NAME || 'insomnia';
```

Comment:
This should not be changed, it refers to the user data directory on disk, not the package name.

```suggestion
process.env.DEFAULT_APP_NAME = process.env.DEFAULT_APP_NAME || 'insomnia-app';
```

Reply:
!!!!!!!!!!!!!! HOLY CRAP GOOD FIND 3eead72366259b84a03082623378d956db6a1560